docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -139,6 +139,63 @@ is used for relative date filtering:
 include-tagged::{sql-specs}/docs/docs.csv-spec[filterToday]
 --------------------------------------------------
 
+[[sql-functions-current-time]]
+==== `CURRENT_TIME/CURTIME`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+CURRENT_TIME
+CURRENT_TIME(precision <1>)
```

Comment:
I thought we were using the `CURRENT_TIME([precision <1>])` (square brackets) for optional parameters, but looking at CURRENT_TIMESTAMP I see the same approach. Other functions (ROUND, TRUNCATE and few string functions) use the square brackets format.

Reply:
Adding the square brackets, will also fix it for CURRENT_TIMESTAMP with a separate PR.